practice-two/src/constants/components.jsx

```diff
@@ -0,0 +1,5 @@
+const BUTTON = {
+  DEFAULT: 'bg-white hover:border-secondary border-2 border-light',

```

Comment:
what is this? Why don't declare it at Button component

Reply:
Fixed it